collections/_events/20250327-project-end.md

```diff
@@ -0,0 +1,14 @@
+---
+title: "HPC Ready - Three years of developing RSE skills with UNIVERSE-HPC (tentative)"
+location: <a href="https://bayes-centre.ed.ac.uk/" target="_blank" rel="noopener noreferer">Bayes Centre</a>, University of Edinburgh and Online
+hybrid: True
+image: "/assets/images/events/20250207_hackathon4-imperial.jpg"
+image_caption: <small>Photo by <a href="https://stocksnap.io/author/4440">Negative Space</a> on <a href="https://stocksnap.io/photo/macbook-computer-6UHHE19YG7">Stocksnap</a></small>"
+date: 27-03-2025 10:00 UTC
+time-range: 10:00-16:00 UTC
+time-range-link: "https://www.timeanddate.com/worldclock/fixedtime.html?msg=HPC+Ready+-+Three+years+of+developing+RSE+skills+with+UNIVERSE-HPC+%28tentative%29&iso=20250327T10&p1=1440&ah=6"
+event_page: 2025-03-27_project-end
+slides: ""
+video: ""
+overview: "This is the UNIVERSE-HPC end of project celebration event. This will be a great opportunity to network with people involved with the project and also the wider RSE and HPC community!<br/><a href=\"https://www.universe-hpc.ac.uk/events/code-of-conduct\" target=\"_blank\">Event code of conduct</a>"
```

Comment:
I'm suggesting a possible rewording here so that it's clear that this is not just a celebration of the project and what we've been doing but that it's also a wider community event with talks from other projects. See what you think and feel free to edit or reject this suggestion.
```suggestion
overview: "This event will mark the end of the UNIVERSE-HPC project. We'll have talks from members of the project as well as the wider RSE and HPC training community. This will be a great opportunity to meet and network with members of the community and find out more about the work going on in this area!<br/><a href=\"https://www.universe-hpc.ac.uk/events/code-of-conduct\" target=\"_blank\">Event code of conduct</a>"
```